### DISCUSSION TITLE
Center for Biologic Diversity v. FAA (CENTER FOR BIOLOGICAL DIVERSITY v. FEDERAL AVIATION ADMINISTRATION (1:23-cv-01204)

### DISCUSSION THREAD

**Discussion by Flowergirlx** [freelawproject/courtlistener#2954] [Q&A] [answered]
For the July 25th docket entry for the above case the information as to the judge's decision for the request for extension by the defendents is missing.
There is no document to view nor any typed in information as with the previous orders giving by the judge.
How do I get CourtListener to update this docket entry for July 25th?

**Comment by mlissner** [comment, marked as answer]
The July 25th entries appear to be minute entries, which means they don't have documents: https://www.courtlistener.com/docket/67303601/center-for-biological-diversity-v-federal-aviation-administration/

Updates usually come when somebody purchases the content using the RECAP extension. If you want an update, install it, click the "Buy on PACER" button, and then buy it from PACER (or use your quarterly free quota).
